dxfeed/adapter.js

```diff
@@ -0,0 +1,51 @@
+const { Requester, Validator } = require('@chainlink/external-adapter')
+const username = process.env.API_USERNAME
+const password = process.env.API_PASSWORD
+const DEFAULT_DATA_ENDPOINT = 'events.json'
+const DEMO_ENDPOINT = 'https://tools.dxfeed.com/webservice/rest'
+const apiEndpoint = process.env.API_ENDPOINT || DEMO_ENDPOINT
```

Comment:
How about we add a warning for operators?

```suggestion
const apiEndpoint = process.env.API_ENDPOINT || DEMO_ENDPOINT
if (apiEndpoint === DEMO_ENDPOINT) console.warn(`Using demo endpoint: ${DEMO_ENDPOINT} (Please do not use in production!)`
```